--- piCorePlayerMeta.lua.orig	2018-10-18 10:29:20.000000000 -0400
+++ piCorePlayerMeta.lua	2019-07-21 14:42:57.014136200 -0400
@@ -26,14 +26,14 @@
 end
 
 function configureApplet(self)
-    local icon
+--  local icon
     local skin = jiveMain:getDefaultSkin()
 
-    if skin == 'JogglerSkin' or skin == 'PiGridSkin' then
-        icon = jiveMain:getSkinParamOrNil('piCorePlayerStyle')
-    else
-        icon = 'hm_settings'
-    end
+--  if skin == 'JogglerSkin' or skin == 'PiGridSkin' then
+--      icon = jiveMain:getSkinParamOrNil('piCorePlayerStyle')
+--  else
+--      icon = 'hm_settings'
+--  end
 
 	-- we only register the menu her, as registerApplet is being called before the skin is initialized
     jiveMain:addItem(
@@ -46,7 +46,7 @@
     		end,
     		100,
     		nil,
-		icon
+		hm_settings
     	)
     )
 
